tests/RelationsTest.php

```diff
@@ -255,36 +260,6 @@ public function testBelongsToMany(): void
         $this->assertCount(1, $client->users);
     }
 
-    public function testSyncBelongsToMany()
```

Comment:
I merged this test with `testBelongsToManySync` test. because I see no reason to keep them separate and I think it's cleaner now.